fuel-tx/src/transaction/validity.rs

```diff
@@ -321,6 +322,24 @@ where
     Ok(())
 }
 
+pub(crate) fn check_owner<T>(tx: &T) -> Result<(), ValidityError>
```

Comment:
Do you think adding some UTs to this function makes sense?

Reply:
I don't really think so it's well tested with the other tests and it's not a practice we currently have for validation rules. I don't find this one more sensible or difficult to understand.